modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -1109,8 +1110,11 @@ private void sendSafeTimeSyncIfReplicaReady(CompletableFuture<Replica> replicaFu
                 .build();
 
         replica.processRequest(req, localNodeId).whenComplete((res, ex) -> {
-            if (ex != null && !hasCauseOrSuppressed(ex, NodeStoppingException.class)
-                    && !hasCauseOrSuppressed(ex, CancellationException.class)) {
+            if (ex != null
+                    && !hasCauseOrSuppressed(ex, NodeStoppingException.class)
+                    && !hasCauseOrSuppressed(ex, CancellationException.class)
+                    && !hasCauseOrSuppressed(ex, RejectedExecutionException.class)
```

Comment:
In what case will RejectedExecutionException be thrown?

Reply:
This change is not related to this PR in particular, this exception sometimes gets thrown on node stop, because some tasks get submitted to a stopped executor. I've seen the same code in some other component, so I copied the approach.